packages/overlay/src/integrations/sentry/components/events/error/Frame.tsx

```diff
@@ -44,26 +49,39 @@ export default function Frame({
   const [isOpen, setOpen] = useState(defaultExpand);
 
   const hasSource = !!frame.context_line;
+  const fileName = platform === 'java' ? frame.module : frame.filename || frame.module;
   return (
     <li className="border-primary-900 bg-primary-900 border-b last:border-b-0">
       <div
         className={classNames(
           hasSource ? 'hover:bg-primary-800 cursor-pointer' : '',
           'border-primary-950 text-primary-400 border-b px-2 py-1',
         )}
-        onClick={() => setOpen(!isOpen)}
+        onClick={hasSource ? () => setOpen(!isOpen) : undefined}
       >
-        {platform === 'java' ? (
-          frame.module ? (
-            <span className="text-primary-100">{`${formatFilename(frame.module)} in `}</span>
-          ) : frame.filename ? (
-            <span className="text-primary-100">{`${formatFilename(frame.filename)} in `}</span>
-          ) : null
-        ) : frame.filename ? (
-          <span className="text-primary-100">{`${formatFilename(frame.filename)} in `}</span>
-        ) : frame.module ? (
-          <span className="text-primary-100">{`${formatFilename(frame.module)} in `}</span>
+        {fileName ? (
+          <span className="text-primary-100">
+            {formatFilename(fileName)}
+            {isOpen && !frame.filename?.includes(':') ? (
+              <PenIcon
+                width={14}
+                height={14}
+                title="Open in editor"
```

Comment:
in order to make this title work.

lets update `svgr()` to 
```
svgr({
  svgrOptions: {
     titleProp: true,
  },
 }),
```

in `packages/overlay/vite.config.ts` under plugins